backend/jest.setup.js

```diff
@@ -90,7 +90,7 @@ function createSocketAgent (cache) {
   return agent
 }
 
-function createAgent (type = 'http', cache) {
+async function createAgent (type = 'http', cache) {
```

Comment:
async declaration not needed here
```suggestion
function createAgent (type = 'http', cache) {
```

Reply:
Hi there @petersutter 

true, I added it for clarity and for streamlining the usage of the `createAgent()` function. 

With adding async in front of it, the function is always returning a promise, which needs to be resolved.  If we omit it, then `createAgent()` returns only sometimes a promise. To be more specific it would return promise if `http` is pased as type and no promise if `io` is parsed as type. depending on the type that is passed as an input param. 

I could imagen this to be confusing to consumers of the function. But I am also happy to adapt to it. 

Looking forward to your opinion 🤗